packages/rs-platform-value/src/display.rs

```diff
@@ -0,0 +1,45 @@
+use crate::Value;
+
+impl Value {
+    pub fn string_representation(&self) -> String {
```

Comment:
shouldn't be implemented `std::fmt::Display` instead?

Reply:
hmmm good point, but I'll do both

Reply:
my brother in Christ our APIs are obese with tons of highly specialized methods and spread across multiple files for a one single type, I truly regret that I used `impl GroveDb` once and now we have large `impl Drive` and now this type becomes huge; having a standard API is a blessing and with `Display` you'll have a `.to_string()` for free, I see not a single reason to have this method

Reply:
okay will remove it from being public then.